pkg/controller/backup_controller.go

```diff
@@ -55,18 +57,22 @@ const backupVersion = 1
 type backupController struct {
 	*genericController
 
-	backupper             backup.Backupper
-	pvProviderExists      bool
-	lister                listers.BackupLister
-	client                arkv1client.BackupsGetter
-	clock                 clock.Clock
-	backupLogLevel        logrus.Level
-	newPluginManager      func(logrus.FieldLogger) plugin.Manager
-	backupTracker         BackupTracker
-	backupLocationLister  listers.BackupStorageLocationLister
-	defaultBackupLocation string
-	metrics               *metrics.ServerMetrics
-	newBackupStore        func(*api.BackupStorageLocation, persistence.ObjectStoreGetter, logrus.FieldLogger) (persistence.BackupStore, error)
+	backupper                    backup.Backupper
+	pvProviderExists             bool
+	lister                       listers.BackupLister
+	client                       arkv1client.BackupsGetter
+	clock                        clock.Clock
+	backupLogLevel               logrus.Level
+	newPluginManager             func(logrus.FieldLogger) plugin.Manager
+	backupTracker                BackupTracker
+	backupLocationLister         listers.BackupStorageLocationLister
+	defaultBackupLocation        string
+	volumeSnapshotLocationLister listers.VolumeSnapshotLocationLister
+	volumeSnapshotLocations      []string
+	metrics                      *metrics.ServerMetrics
+	newBackupStore               func(*api.BackupStorageLocation, persistence.ObjectStoreGetter, logrus.FieldLogger) (persistence.BackupStore, error)
+	// TODO: figure out how to create a new volume
+	newVolume func(*api.VolumeSnapshotLocation, logrus.FieldLogger) (persistence.BackupStore, error)
```

Comment:
I *think* we'll be instantiating block stores within `pkg/backup` (which I'm guessing is what this addition was eventually intended to do) - so I don't think this is needed here